.github/workflows/pr.yml

```diff
@@ -30,23 +31,18 @@ jobs:
           tagFormat: v${version}
         id: version_check_master
   build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [windows-latest, macos-latest, ubuntu-latest]
-        node:
-          - stable
-          - lts
-      fail-fast: false
+    runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v2
 
-      - name: Install dependencies
-        uses: actions-rs/cargo@v1
-        with:
-          command: npm ci
-      
-      - name: Publish to npm
-        uses: actions-rs/cargo@v1
+      - name: Setup nodejs
+        uses: actions/setup-node@v1
         with:
-          command: npm publish
+          node-version: '12.x'
```

Comment:
`npm run build`, `npm run pack`, and upload the tar artifact from this job too, just in case we need to download a tarball from a specific build for debugging.

Reply:
This too?

Reply:
Yep